plugins/com.google.cloud.tools.eclipse.javax.annotations/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
+org.eclipse.jdt.core.compiler.compliance=1.8
+org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
+org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
+org.eclipse.jdt.core.compiler.source=1.8
```

Comment:
Is 1.8 intended? After opening this javax.annotation project (module, bundle, or whatever), my eclipse seems to change these to 1.7. (Maybe after I set to use JavaSE-1.7?)